app/views/documents/show/_scheduling_form.html.erb

```diff
@@ -0,0 +1,52 @@
+<% publish_datetime = @edition.revision.scheduled_publishing_datetime %>
```

Comment:
Minor, but could we do the `||` up here, to simplify the `||`s down below? I guess there's a compromise with the two other checks that involve this variable on its own, but I think it'll help readability if we make the following lines a bit simpler :-).